### PR TITLE
v2.45 uk l10n

### PR DESCRIPTION
**Pull request by arkid15r**
v2.45 uk update

### CODE REVIEW

File: po/uk.po [hunk 1]
```diff
@@ -1918,14 +1921,8 @@ msgstr ""
...
-msgid ""
-"Use -f if you really want to add them.\n"
-"Turn this message off by running\n"
-"\"git config advice.addIgnoredFile false\""
-msgstr ""
-"Використовуйте -f, якщо ви дійсно хочете їх додати.\n"
-"Щоб вимкнути це повідомлення, виконайте\n"
-"\"git config advice.addIgnoredFile false\""
+msgid "Use -f if you really want to add them."
+msgstr "Скористайтесь -f, якщо ви дійсно хочете їх додати"
```
Comment (kasya) [inline po/uk.po]:
```suggestion
msgstr "Скористайтесь -f, якщо ви дійсно хочете їх додати."
```

File: po/uk.po [hunk 2]
```diff
@@ -11070,6 +11075,9 @@ msgstr ""
 msgid "could not start pack-objects to repack promisor objects"
 msgstr "не вдалося розпочати pack-objects для перепакування promisor обʼєктів"
 
+msgid "failed to feed promisor objects to pack-objects"
+msgstr "не вдалося передати promisor обʼєкти pack обʼєктам"
```
Comment (kasya) [inline po/uk.po]: 🤔 
```suggestion
msgstr "не вдалося передати promisor обʼєкти обʼєктам пакунків"
```

File: po/uk.po [hunk 3]
```diff
@@ -15508,7 +15539,7 @@ msgstr "ssh опція \"simple\" не підтримує встановленн
 
 #, c-format
 msgid "strange pathname '%s' blocked"
-msgstr "дивне імʼя шляху \"%s\" заблоковано"
+msgstr "дивна назва шляху \"%s\" заблоковано"
```
Comment (kasya) [inline po/uk.po]:
```suggestion
msgstr "дивна назва шляху \"%s\" заблокована"
```

File: po/uk.po [hunk 4]
```diff
@@ -19390,6 +19460,79 @@ msgstr "не вдалося видалити посилання %s: %s"
...
+msgid "reftable: transaction prepare: %s"
+msgstr "таблиця посилань: підготовка транзакції: \"%s\""
+
+#, c-format
+msgid "reftable: transaction failure: %s"
+msgstr "таблиця посилань: помилка транзакції: \"%s\""
+
+#, c-format
+msgid "unable to compact stack: %s"
+msgstr "не вдалося ущільнити стек: %s"
```
Comment (kasya) [inline po/uk.po]: Not sure if my suggestion is better... 

```suggestion
msgstr "не вдалося стиснути стек: %s"
```

**Review by kasya — COMMENTED** [review]
👍🏼

File: po/uk.po (hunk 1 above)
Comment (arkid15r) [reply po/uk.po]: 💯

**Review by arkid15r — COMMENTED** [review]
👍 thanks for the review!

**Pull request closed without merging**